CHANGELOG.md

```diff
@@ -5,23 +5,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## In-development
 
-- Fixed nf-core tools linting failures introduced in version 2.12.1.
+- Fixed nf-core tools linting failures introduced in version 2.14.1.
 - Added phac-nml prefix to nf-core config
+- Added ternary conditional operator in container section of process directive for singularity containers
```

Comment:
Could you add a reference to the pipeline-standards here which has more information: https://github.com/phac-nml/pipeline-standards?tab=readme-ov-file#522-configuring-module-software-with-private-or-alternate-container-registries

Reply:
I elaborated on this a bit. 911d167